### PR TITLE
Feat: add legal text

### PR DESCRIPTION
**Pull request by begonaalvarezd**
# Description of change

Add legal text

## Links to any relevant issues

N/A

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Ran on Ubuntu 18.04

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/routes/setup/Legal.svelte [hunk 1]
```diff
@@ -53,26 +48,695 @@
...
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    The App must be downloaded and saved to your desktop device in order for you to use the App;
+                                </Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    Crypto-asset transactions are irreversible. As the User, it is your responsibility to ensure
+                                    that any crypto-assets that you send with the App are sent to the correct address. We do not
+                                    have access to the wallets created through the App and we are not be able to assist you in
```
Comment (obany) [inline packages/shared/routes/setup/Legal.svelte:179]: this doesn't look correct `and we are not be able`

File: packages/shared/routes/setup/Legal.svelte [hunk 2]
```diff
@@ -53,26 +48,695 @@
...
+                                    from force majeure, riot, war and natural events, or from any other events for which it cannot
+                                    be held responsible (e.g. strike, lockout, traffic disruptions, orders proclaimed by a supreme
+                                    authority).
+                                </Text>
+                            </li>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    The IOTA Foundation does not accept any liability for transactions, due to a lack of control
+                                    over transaction processes. We are therefore expressly not responsible for any claims, damages
+                                    or any other type of liability caused by any third-party service provider.In no event will the
```
Comment (obany) [inline packages/shared/routes/setup/Legal.svelte:286]: Missing space before `.In no event`

Comment (Thoralf-M) [inline packages/shared/routes/setup/Legal.svelte:286]: Missing space `provider.In`

File: packages/shared/routes/setup/Legal.svelte [hunk 3]
```diff
@@ -53,26 +48,695 @@
...
+                    </Text>
+                    <Text type="h3" classes="mb-5">CoinGecko</Text>
+                    <Text type="p" secondary classes="mb-4">
+                        We use CoinGeckoAPI of Gecko Labs Pte. Ltd, 101 Upper Cross Street, #05-16 People's Park Centre, Singapore
+                        058357 ("CoinGecko") to obtain historical price and market data for cryptocurrencies. Further information
+                        regarding their privacy policy can be found under https://www.coingecko.com/en/privacy
+                    </Text>
+                    <Text type="h3" classes="mb-5">GitHub</Text>
+                    <Text type="p" secondary classes="mb-4">
+                        We use GitHub of GitHub Inc 88 Colin P. Kelly Street, San Francisco, CA 94107, USA ("GitHub" - in the
```
Comment (obany) [inline packages/shared/routes/setup/Legal.svelte:597]: We use S3 for auto update not GitHub

File: packages/shared/routes/setup/Legal.svelte [hunk 4]
```diff
@@ -53,26 +48,695 @@
...
+                        identifiable natural person is one who can be identified, directly or indirectly, in particular by
+                        reference to an identifier such as a name, an identification number, location data, an online identifier
+                        or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural or
+                        social identity of that natural person.
+                    </Text>
+
+                    <Text type="h2" classes="mt-8 mb-5">How we Collect Personal Data</Text>
+                    <Text type="p" secondary classes="mb-4">
+                        IOTA does not collect your Personal Data through the App. However, if you download, install and use the
+                        App on your phone or other device, it may be necessary to process some data about you or your device for
```
Comment (obany) [inline packages/shared/routes/setup/Legal.svelte:468]: Can't use this version on phone

File: packages/shared/routes/setup/Legal.svelte [hunk 5]
```diff
@@ -53,26 +48,695 @@
...
+                                    we have a legitimate interest in carrying out the processing for the purpose of managing,
+                                    operating or promoting our business, and that legitimate interest is not overridden by your
+                                    interests, fundamental rights, or freedoms.
+                                </Text>
+                            </li>
+                        </ul>
+                    </Text>
+
+                    <Text type="h2" classes="mt-8 mb-5">Use of Cookies</Text>
+                    <Text type="p" secondary classes="mb-4">We do not use cookies in the App.</Text>
```
Comment (obany) [inline packages/shared/routes/setup/Legal.svelte:546]: But we do use localStorage which is included in the same `cookie` laws

https://law.stackexchange.com/questions/30739/do-the-gdpr-and-cookie-law-regulations-apply-to-localstorage

**Review by cvarley100 — APPROVED** [review]

File: packages/shared/routes/setup/Legal.svelte [hunk 6]
```diff
@@ -53,26 +48,695 @@
...
+                        purchase of crypto-assets shall be the sole responsibility of the User.
+                    </Text>
+
+                    <Text type="h2" classes="mt-8 mb-5">6. Right of Withdrawal and Termination</Text>
+                    <Text type="p" secondary classes="mb-4">
+                        <ul>
+                            <li>
+                                <Text type="p" secondary classes="mb-4">
+                                    By entering into this Agreement you hereby waive your right of withdrawal from it. You may
+                                    simply stop using it.
```
Comment (rajivshah3) [inline packages/shared/routes/setup/Legal.svelte:224]:
```suggestion
                                    simply stop using the App.
```

**Pull request merged**